people.json

```diff
@@ -0,0 +1 @@
+[{"name":3,"age":"Unknown","permission":"Raslaan","type":"Student"}]
```

Comment:
- [ ] Gead job so far however you still have some issue with the people.json file, actually, student or teacher information are not saved correct, have a look at the screenshot 👇🏻 

![image](https://user-images.githubusercontent.com/79058364/217603142-e0e9e13a-b36a-42ca-b697-fbd19ba62028.png)

- The first case the name is a number instead of a string and the age has `Unknown` as a value.
- The second case and the age always has an `Unknown` value even when you give the name to the student.

- [ ] Kindly fix this and make sure everything works as expected 💯 👍🏻